chain/chain/src/chain.rs

```diff
@@ -1527,19 +1530,21 @@ impl Chain {
         if shard_id as usize >= sync_prev_block.chunks().len() {
             return Err(ErrorKind::InvalidStateRequest("shard_id out of bounds".into()).into());
         }
+        // TODO: why is this prev_state_root here?
```

Comment:
what is the action item here?

Reply:
sorry, that was me commenting when I read code. Will delete this